### DISCUSSION TITLE
段落超出显示...

### DISCUSSION THREAD

**Discussion by lrx1994** [openkraken/kraken#720] [General]
我自己在deom中尝试了,设置两行后,段落还是没有换行,直接一行显示的,直接超出屏幕了,这是什么问题呢?

**Comment by wssgcg1213** [comment]
可以把节点结构以及样式提供一下吗? 我们可以本地复现下

**Comment by lrx1994** [comment]
`<div style={{
                display:"flex",
                justifyContent:"space-between",
                padding:'15px'
            }}>
                <div style={{
                    width:'80px',
                    height:'80px',
                    backgroundColor:'pink',
                    borderRadius:'8px'
                }}>
                </div>
                <div style={{
                    fontSize: '15px',
                    color: '#333333',
                    textOverflow:'ellipsis',
                    lineClamp:2,
                    flex:1,
                    marginLeft:'15px'
                }}>
                    南山经之首，曰鹊山。其首曰招摇之山，临于西海之上， 多桂，多金玉。有草焉，其状如韭而青华，其名曰祝余，食之不饥。有木焉，其状如榖而黑理，其华四照，其名曰迷榖，佩之不迷。有兽焉，其状如禺而白耳，伏行人走，其名曰狌狌，食之善走。丽麂之水出焉，而西流注于海，其中多育沛，佩之无瘕疾
                </div>
            </div>`
![微信图片_20211009175829](https://user-images.githubusercontent.com/24952476/136653609-a3612889-049e-47aa-b577-9f9faf675772.jpg)
我这样写超出屏幕,我这里必须写固定宽度么?谢谢您答复
